classes/submission/reviewer/form/ReviewerReviewForm.php

```diff
@@ -113,20 +107,18 @@ public function fetch($request, $template = null, $display = false)
      * Set the review step of the submission to the given
      * value if it is not already set to a higher value. Then
      * update the given reviewer submission.
-     *
-     * @param ReviewerSubmission $reviewerSubmission
      */
-    public function updateReviewStepAndSaveSubmission($reviewerSubmission)
+    public function updateReviewStepAndSaveSubmission(ReviewAssignment $reviewAssignment)
     {
         // Update the review step.
         $nextStep = $this->getStep() + 1;
-        if ($reviewerSubmission->getStep() < $nextStep) {
-            $reviewerSubmission->setStep($nextStep);
+        if ($reviewAssignment->getData('step') < $nextStep) {
+            $reviewAssignment->setData('step', $nextStep);
```

Comment:
I think you can use your new helper methods to `getStep()` and `setStep()`.

Reply:
Fixed